applications/dashboard/models/class.usermodel.php

```diff
@@ -4921,4 +4914,61 @@ public function clearSectionNavigationPreference($url = '', $userID = '', $reset
             $this->savePreference($userID, 'DashboardNav.DashboardLandingPage', '');
         }
     }
+
+    /**
+     * @param int $userID
+     * @throws Exception
+     * @return bool
+     */
+    public function isApplicant($userID) {
+        $result = false;
+
+        $applicantRoleIDs = RoleModel::getDefaultRoles(RoleModel::TYPE_APPLICANT);
+
+        // Make sure the user is an applicant.
+        $roleData = $this->getRoles($userID);
+        if ($roleData->numRows() == 0) {
```

Comment:
You can call `count()` on a Gdn_DataSet.
`class Gdn_DataSet implements IteratorAggregate, Countable, JsonSerializable`